features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/base64/Base64Serializer.kt

```diff
@@ -270,23 +273,38 @@ internal class Base64Serializer private constructor(
         imageWireframe: MobileSegment.Wireframe.ImageWireframe,
         base64SerializerCallback: Base64SerializerCallback
     ): String? {
-        return base64LRUCache?.get(drawable)?.let { base64String ->
-            finalizeRecordedDataItem(base64String, imageWireframe, base64SerializerCallback)
-            base64String
+        val encodings = base64LRUCache.get(drawable)
```

Comment:
`resourceId` cannot be classified as encoding, so better to use other name.

```suggestion
        val cacheData = base64LRUCache.get(drawable)
```

Reply:
👍  I'll change in all relevant places